### DISCUSSION TITLE
Ideas for greater privacy

### DISCUSSION THREAD

**Discussion by menny** [AnySoftKeyboard/AnySoftKeyboard#2656] [Ideas]
If you have any ideas about how AnySoftKeyboard can help you keep your privacy, please let us know.

For example, https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/2630

**Comment by alchemiker** [comment]
This bug #2784 showing plain passwords in clipboard might be a privacy issue.

**Comment by adeyo50** [comment]
The microphone button seems to force use Google TTS engine by default. I've added flite TTS engine to my phone and set it as default but Google is still used for some reason. Maybe make this a choice in settings somehow?  Obviously using google is a major data leak
